maintenance/wikia/fixes/fixVisualizationImages.php

```diff
@@ -54,7 +54,7 @@ protected function getWikisToFix() {
 	protected function getCVImage( $wikiId ) {
 		global $wgExternalSharedDB;
 		$dbr = wfGetDB( DB_SLAVE, array(), $wgExternalSharedDB );
-		$result = $dbr->selectRow( array( CityVisualization::CITY_VISUALIZATION_IMAGES_TABLE_NAME ), array( '*' ), array( 'city_id' => $wikiId, 'image_index' => 0, 'image_review_status' => ImageReviewStatuses::STATE_APPROVED ), __METHOD__ );
+		$result = $dbr->selectRow( array( CityVisualization::CITY_VISUALIZATION_IMAGES_TABLE_NAME ), array( '*' ), array( 'city_id' => $wikiId, 'image_index' => 0, 'image_review_status' => ImageReviewStates::APPROVED ), __METHOD__ );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$result". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)